theme/functions.php

```diff
@@ -28,10 +28,22 @@ function mytheme_setup_theme_supported_features() {
 
 	// Define a palette
 	add_theme_support( 'editor-color-palette',
-			'#334499',
-			'#d0a5db',
-			'#eee',
-			'#444'
+		array(
+			'name' => 'strong blue', // theme should define has-strong-blue-background-color and has-strong-blue-color
```

Comment:
Nitpick, but: how can a colour be “strong”? I can picture a “deep blue” or “rich blue” but not so much “strong”. 🤷🏻‍♂️

Reply:
In this case, I think I did a mistake because strong blue is used in Gutenberg starter theme for another color https://github.com/WordPress/gutenberg-starter-theme/blob/b0b9feeef8babaed5f55d132164ae3c03a34976f/functions.php#L91-L94. We should also probably also update the name there.